packages/js/src/ui/components/elements/Footer.tsx

```diff
@@ -3,22 +3,48 @@ import { Show } from 'solid-js';
 import { useInboxContext } from 'src/ui/context';
 import { isBrowser } from 'src/utils/is-browser';
 import { Novu } from '../../icons';
+import { cn } from '../../helpers';
+
+const stripes = `before:nt-content-[""] before:nt-absolute before:nt-inset-0 before:-nt-right-[calc(0+var(--stripes-size))] before:nt-bg-dev-stripes-gradient before:nt-bg-[length:var(--stripes-size)_var(--stripes-size)] before:hover:nt-animate-stripes`;
+const commonAfter = 'after:nt-content-[""] after:nt-absolute after:nt-inset-0 after:-nt-top-8';
+const devModeGradient = `${commonAfter} after:nt-bg-[linear-gradient(180deg,#0000_0,#FFF6F3_50%,#0000)]`;
```

Comment:
this `#FFF6F3` is the color of the gradient background that is rendered on top of stripes

Reply:
I don't think that we want to expose it as a variable

Reply:
I agree.